application/F3DOptionsParser.cxx

```diff
@@ -289,11 +289,10 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
   try
   {
     cxxopts::Options cxxOptions(this->ExecutableName, F3D::AppTitle);
-    cxxOptions.positional_help("file1 file2 ...");
+    cxxOptions.custom_help("[OPTIONS...] file1 file2 ...");
```

Comment:
what does this do ?

Reply:
related test is broken btw

Reply:
`positional_help` has no effect when there's no positional arguments so I used `custom_help` to mimic the behavior.
What do you mean about the test?

Reply:
if you check the CI (or run tests locally) you will see:

```
Errors while running CTest
Output from these tests are in: /__w/f3d/f3d/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
1/1 Test #372: f3d::TestHelpPositional ..........***Failed  Required regular expression not found. Regex=[--input
]  0.04 sec

```

It is about a regex that check for the presence of `input`, which obviously is not here anymore.